composer/modules/web/src/plugins/ballerina/diagram/views/default/components/decorators/http-resource-header.jsx

```diff
@@ -189,4 +189,4 @@ HttpResourceHeader.propTypes = {
     y: PropTypes.number.isRequired,
 };
 
-export default HttpResourceHeader;
\ No newline at end of file
+export default HttpResourceHeader;
```

Comment:
Composer folks are ok with this right ?

Reply:
New line after this statement gets automatically removed from Idea. So added it back as shown below.

Reply:
@shafreenAnfar ; If there is a new line after export, then its fine for our esLint checks 👍